llvm/include/llvm/ADT/STLExtras.h

```diff
@@ -1994,6 +1994,10 @@ auto unique(Range &&R, Predicate P) {
   return std::unique(adl_begin(R), adl_end(R), P);
 }
 
+template <typename Range> auto unique(Range &&R) {
```

Comment:
Please make this change (moving `unique(Range&&)` to STLExtras, and adding unit test coverage) in a separate commit (per https://llvm.org/docs/DeveloperPolicy.html#incremental-development )

Reply:
Done: https://github.com/llvm/llvm-project/pull/82312